websites/A/Anime Genzai/metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Anime Genzai",
+  "author": {
+    "id": "257151906038677504",
+    "name": "SHISEN#8023"
+  },
+  "description": {
+    "en": "Discover the best subtitles and dubbed Anime series and movies with advanced filters and watch them in HD."
+  },
+  "url": "animegenzai.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/dtmYEl8.png",
+  "thumbnail": "https://i.imgur.com/irueb07.png",
+  "color": "#f5c518",
+  "tags": [
+    "animegenzai",
+    "anime"
+  ],
+	"iframe": true,
+  "category": "anime",
+  "settings": [
+		{
+			"id": "lang",
```

Comment:
This file is not formatted correctly. Use `yarn format` to format this file.

Reply:
Oh... I forgot about it.